updating Creator user doc

jakubjezek001 opened the pull request:
added info pane with current Publisher / Create tab image

mkolar reviewed (changes requested):
I have a feeling that we should rather keep the creator as is, but putting a notice that it's being deprecated and replaced by the create tab in publisher.

On this page they would see how to use the old creator and would just have a link to the new publisher that should document all the tabs. 

@jakubjezek001 what do you think? Right now this actually takes away information that is still relevant in many hosts.

jakubjezek001 commented:
> @jakubjezek001 what do you think? Right now this actually takes away information that is still relevant in many hosts.

I had moved the order for the info panel and added old creator image to make it more obvious.

mkolar commented on website/docs/artist_tools_creator.md:
```diff
@@ -15,11 +17,17 @@ In Maya this means creating a set with everything you want to publish and assign
 In Nuke it's either converting an existing write node to a publishable one, or simply creating a write node with all the correct settings and outputs already set.
 
+:::info
+Current Publisher is containing Creator as tab.
```
```suggestion
Current Publisher contains Creator as tab.
```
